README.adoc

```diff
@@ -288,6 +287,7 @@ Next, this test increases the priority of the custom configuration source and sw
 In the end, the inventory service returns the following message: `ERROR: Service is currently in maintenance`.
 
 In addition, a few endpoint tests have been provided for you to test the basic functionality of the `inventory` and `system` services. If a test failure occurs, then you must have introduced a bug into the code.
+It is crucial to register the custom configuration source and custom converter in the `start/src/main/resources/META-INF/services/` directory, if you miss to complete these steps, the tests will fail.
```

Comment:
This is a repeat of the above.  Perhaps we can say something like "Note that ..." or "Reminder that ...".

Reach out to @sishida for help if necessary.